run_app.py

```diff
@@ -1,7 +1,38 @@
-from streamlit.web import cli
+import argparse
+import json
+import subprocess
+import logging
+
+# Configure logging
+logging.basicConfig(level=logging.WARNING)
+
+# Default upload size (MB) if settings.json is missing/invalid
+DEFAULT_MAX_SIZE = 200
```

Comment:
Might be better to have this in a new file constants.py so all the constants are in one place and import it from there.